src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -264,10 +266,8 @@ async def patch(self):
             schema:
               $ref: '#/definitions/Patch'
         responses:
-          200:
-            description: Execute Patch action against Gardens
-            schema:
-              $ref: '#/definitions/Garden'
+          204:
```

Comment:
Despite what this said before, it seemed like the "sync" operation didn't return any content.  Neither does the "sync_users" operation, so I just changed this to a 204, which is more appropriate.